gear/gear/metrics.py

```diff
@@ -25,3 +28,38 @@ async def monitor_endpoints_middleware(request, handler):
         raise e
     finally:
         CONCURRENT_REQUESTS.labels(endpoint=endpoint, verb=verb).dec()
+
+
+class LoggingTimer:
+    def __init__(self, query_name, threshold_ms=None):
+        self.query_name = query_name
+        self.threshold_ms = threshold_ms
+        self.timing = {}
+        self.start_time = None
+        self.sql_query_latency_manager = None
+
+    def step(self, name):
+        return LoggingTimerStep(self, name)
```

Comment:
Given our refactor, this method is probably unused. Can you make sure that it is and then delete it? Along with `LoggingTimerStep`